terragrunt/modules/docs-rs/builder.tf

```diff
@@ -32,7 +32,14 @@ resource "aws_iam_role_policy" "builder_s3" {
       // Access to s3
       {
         Effect = "Allow"
-        Action = "s3:*"
+        Action = [
+          "s3:PutObject",
+          "s3:GetObject",
+          "s3:CreateBucket",
```

Comment:
we shouldn't need CreateBucket I don't think?

Reply:
I believe that it the bucket doesn't exist (albeit an unlikely possibility), the worker can create the bucket.

Reply:
We only ever create a bucket in tests, not in prod: https://github.com/rust-lang/docs.rs/blob/d97bbf54b25362e1d1010f7fb6e29c5324cfb0a6/src/storage/s3.rs#L58

Reply:
I don't think we should grant the builder create bucket in any case, it's too easy to get that wrong - and the bucket is then not controlled by terraform, which also seems bad.

Granting delete bucket is also hopefully not necessary, since it'll be hard to avoid deleting production bucket if things aren't configured right or whatever.

Reply:
Unfortunately [deleting is currently necessary](https://github.com/rust-lang/simpleinfra/issues/238#issuecomment-1427980920).

Reply:
Removed CreateBucket

Reply:
I was specifically referring to delete bucket, not object, but yeah, looks good.